src/test/java/seedu/address/testutil/TypicalRooms.java

```diff
@@ -1,25 +1,54 @@
 package seedu.address.testutil;
 
-import java.util.List;
-
+import seedu.address.model.room.DoubleRoom;
 import seedu.address.model.room.Room;
+import seedu.address.model.room.RoomNumber;
+import seedu.address.model.room.SingleRoom;
+import seedu.address.model.room.SuiteRoom;
 import seedu.address.model.room.UniqueRoomList;
-import seedu.address.model.util.SampleDataUtil;
 
 /**
  * A utility class containing a list of {@code Room} objects to be used in tests.
  */
 public class TypicalRooms {
+
+    private static final int CAPACITY_SINGLE_ROOM = SingleRoom.CAPACITY_SINGLE_ROOM.getValue();
```

Comment:
Actually I think this double calling of methods is not encouraged. The `getValue()` method belongs to the `Capacity` class.

To rectify, can make `Capacity` field in `SingleRoom` private, and instead create a public method `getCapacityValue()`.

Reply:
Fixed.